homeassistant/components/tuya/const.py

```diff
@@ -254,6 +254,7 @@ class DPCode(StrEnum):
     PM25_STATE = "pm25_state"
     PM25_VALUE = "pm25_value"
     POWDER_SET = "powder_set"  # Powder
+    POWER_SWITCH = "Power"
```

Comment:
This seems to be device/manufacturer specific, as this is not a normal DP name.  (capitalized).

Please note, that we don't want to add/implement device specifics, but only use Tuya standardized data points. Have you contacted Tuya about this?

Reply:
Yes TuYa, have been contacted. You have the result here: https://github.com/home-assistant/core/issues/78815#issuecomment-1415277555
The contact of TuYa is the origin of this PR. Devices have 2 modes: standard instruction set or original DP instruction. This PR is made to make the "original TP instruction set" works.

Reply:
@frenck  is this a blocker point ? If yes I will not try to fix your suggestion below and stop this PR.